### ISSUE TITLE
🐜 Accessibility Bug in the toggle/slider for Province/Territory drop down in the About Me Form

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#5411]
## ❌ Current behaviour (issue)
Difficult to use the toggle or slider while trying to select the province. The arrow keys will read the province going down, but not going up.

### Location: 
About Me UI

###AT Configuration
VoiceOver, MacOS, Safari

## 💿 How to reproduce
1. Navigate to **About Me** section from the Application profile with VoiceOver and keyboard navigation. 
2. Activate the the **Current province or territory** drop down. 
3. Arrow down to the bottom of the list. 
4. Arrow up the list from the bottom to confirm that the slider works in MacOS and Safari. 

### Screen Capture
![#4567_Screen Capture.png](https://images.zenhubusercontent.com/625076b8f67c446dd188ffe5/83b755b3-8184-475a-ada3-33cffab33813)

## ✔ Expected behaviour (fix)
The arrow keys will read the province going up.

### Acceptance Criteria
A set of assumptions which, when tested, verify that the accessibility requirement was met.
- [x] Arrow keys can move the focus going down or up the list
- [x] First letter navigation is maintained

**Comment by substrae** [comment]
I don't have MacOS, so I can't test this myself, but I did confirm that it's working fine in Windows on Firefox and Edge. @esizer or @mnigh can either of you test on a Mac?

**Comment by esizer** [comment]
> I don't have MacOS, so I can't test this myself, but I did confirm that it's working fine in Windows on Firefox and Edge. @esizer or @mnigh can either of you test on a Mac?

Hmm, Do we have version numbers? I am unable to reproduce in:

- **Safari**: 16.2 (18614.3.7.1.5)
- **macOS**: 13.1 (22C65)


https://user-images.githubusercontent.com/4127998/213567432-26615a84-67e4-48fc-a326-5bc2e472b0b8.mov

**Comment by mnigh** [comment]
I am unable to reproduce in:

- **Safari**: 16.2 (17614.3.7.1.7)
- **macOS**: 12.6.2 (21G320)

## Screen Recording
https://user-images.githubusercontent.com/3046459/213572059-b4cf54ae-efc0-4bcc-8fff-4ccaaaa7a775.mov

**Comment by RM-1978** [comment]
Alright! Let's close this ticket :)
